src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -152,6 +216,63 @@ const PathWayMapperContainer: React.FunctionComponent<IPathwayMapperContainerPro
                             height={800}
                             url={`${resultsViewPageStore.remoteNdexUrl.result}`}
                         />
+                    </MSKTab>*/}
+                    <MSKTab
+                        key={ResultsViewPathwaysSubTab.NDEX}
+                        id={ResultsViewPathwaysSubTab.NDEX}
+                        linkText={ResultsViewPathwaysSubTab.NDEX}
+                        linkTooltip={
+                            <div style={{ maxWidth: 400 }}>
```

Comment:
you should wrap this whole tooltip thing inside of a component and put the store and remoteData fetching inside of it.  that way it's lifecycle is independent of the tab, which shouldn't care or know about the tooltip (except to render it).    
```
linkTooltip=<TooltipComponent ...props-if-needed />
```